bin/rector.php

```diff
@@ -117,15 +117,17 @@ public function autoloadFromCommandLine(): void
 
     public function loadIfExistsAndNotLoadedYet(string $filePath): void
     {
-        if (! file_exists($filePath)) {
+        $filePath = realpath($filePath);
```

Comment:
I don't know if Rector ever uses VFS for testing, but `realpath()` always fails on virtual paths.

Reply:
I will rollback it as it cause error back on e2e test which look up from parent `../../bin` , probably need check if file path not contains `..`

Reply:
@MGatner rolled back and check parent https://github.com/rectorphp/rector-src/pull/1692/commits/619e987538f29d145c861f540047b90e6f781720

Please verify again, I tried verify manually at my local dev and it seems working ok